bin/detail/call.py

```diff
@@ -63,15 +63,16 @@ def call(call_args, logging, cache_file='', ignore=False, sleep=0):
   for i in call_args:
     pretty += '  `{}`\n'.format(i)
   pretty += ']\n'
-  print(pretty)
+  if logging.verbosity == 'full':
+    print(pretty)
   logging.log_file.write(pretty)
 
   # print one line version
   oneline = ''
   for i in call_args:
     oneline += ' "{}"'.format(i)
   oneline = "[{}]>{}\n".format(os.getcwd(), oneline)
-  if logging.verbose:
+  if logging.verbosity == 'full':
```

Comment:
`oneline` may be helpful in case you want to repeat the steps without Polly (e.g. file a bug without introducing Polly extra layer). In fact I don't use it very often, so I'm okay with `verbosity == 'full'`. However printing CMake commands is a good sanity checking (see line 66 comment).